src/styles/cardStyles.scss

```
@@ -0,0 +1,108 @@
+.text-Main {
+  transition-duration: $duration;
+  color: $Main-accent;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 0 0;
+}
+
+.text-Secondary {
+  color: $Secondary;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 4px 0 0;
```

apply similar recommendations all over the page
```suggestion
  margin-top: 4px;
```